src/terra-dev-site/guide/Component Guides/RecreatingItemCollection.guide.mdx

```diff
@@ -0,0 +1,201 @@
+```
+
+Cells are simple and just require a key, the children of said cell and then we use `attrs` to set `className` in our example like so.
+
+```javascript
+const attrs = { className: cx('content-display')};
+```
+
+## Creating a Cell Grid that looks like a Table
+
+Instead of using `<Table>`, you can use [CellGrid](/components/terra-cell-grid/cell-grid/cell-grid) to create another version of the default content when over a certain window size. This method requires a little bit of CSS to be put in place to get a closer match to `<Table>`'s default style. The basic content look of it looks like this.
+
+```javascript
+  const cellGridDisplay = (
+    <div className={cx('cellgrid-wrapper')}>
+      <CellGrid className={cx('row')}>
+        <Cell className={cx('start-accessory')} key="cell-0" width={{ static: { value: 40, unit: 'px' } }}>{startAccessory}</Cell>
+        <Cell className={cx('display')} key="cell-i" width={{ scalar: 1 }}>{display}</Cell>
+        ...
+      </CellGrid>
+      ...
+    </div>
+  );
+```
+
+In this example, the `<CellGrid>` effectively act as rows and doesn't require any props other than defining a class for it. The `<Cell>`'s are where the content is placed. In the example provided at the bottom of this page, the content has two accessories on either side of the main information. Setting these accessories `width` prop to `{ static: { value: 40, unit: 'px' } }` allows the accessories to only take up a small amount of space, while the rest of the `Cell`'s use `{ scalar: 1 }` for width to automatically take up an equal percentage of the remaining space.
+
+For the main styles it uses the following CSS.
+
+```css
+.cellgrid-wrapper {
+  border-bottom: 1px solid #c8cacb;
+}
+
+.row {
+  border-top: 1px solid #c8cacb;
+  display: flex;
+  min-height: 60px;
+}
+
+.row:nth-child(even) {
+  background: #f4f4f4;
+}
+```
+
+## Creating a read-only example using HTML table
+
+If you are making a read-only table, we suggest using [HTML Table](/components/terra-html-table/html-table/table) which gets put together like this.
+
+```javascript
+const htmlTable = (
+  <Table>
+    <Header>
+      <HeaderCell key="cell-date">{"Date"}</HeaderCell>
+      <HeaderCell key="cell-1">{"Chrome"}</HeaderCell>
+      ...
+    </Header>
+    <Body>
+      <Row key="row_0">
+        <Cell key="display_0" className={cx('cell-date')}>{"October 2019"}</Cell>
+        <Cell key="display_1" className={cx('cell')}>{"68.91%"}</Cell>
+        ...
+      </Row>
+      ...
+    </Body>
+  </Table>
+);
+```
+
+It uses a straightforward Header and Body section. The header section is composed of `<HeaderCell>`'s which just needs a simple key and child for its display.
+
+The body `<Row>`s and `<Cell>`s are also for the most part easily constructed with keys required for both. The `<Cell>`s also get classes for ease of styling and children for actual content.
+
+## Example
+<ResponsiveListandTableExample title='Responsive List And Table Example' />
```

Comment:
Intersperse these examples after each section where you talk about building them to break up the article.

Reply:
No longer applicable since other examples have been removed.